projects.go

```diff
@@ -1512,3 +1512,34 @@ func (s *ProjectsService) StartMirroringProject(pid interface{}, options ...Opti
 
 	return resp, err
 }
+
+// TransferProjectOptions represents the available TransferProject() options.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/projects.html#transfer-a-project-to-a-new-namespace
+type TransferProjectOptions struct {
+	NamespaceID *int `url:"namespace,omitempty" json:"namespace,omitempty"`
```

Comment:
```suggestion
	Namespace interface{} `url:"namespace,omitempty" json:"namespace,omitempty"`
```

Reply:
Good catch, I learned from this that the interface{} type is a better solution for this. I tested this in my self-hosted instance and was able to pass both int/string types (per Gitlab docs of supported types) and it worked for me on v12.6.0.